src/lib/connectors/jsdom/jsdom.ts

```diff
@@ -215,6 +220,22 @@ class JSDOMConnector implements IConnector {
         }
     }
 
+    /** JSDOM doesn't download the favicon automatically, this method:
+     *
+     * * uses the `src` attribute of `<link rel="icon">` if present.
+     * * uses `favicon.ico` and the final url after redirects.
+     */
+    private async getFavicon(element?: HTMLElement) {
+        const href = element ? element.getAttribute('href') : '/favicon.ico';
+
+        //TODO: remove "as any" once https://github.com/DefinitelyTyped/DefinitelyTyped/issues/16860 is fixed
```

Comment:
With the latest package update I think this is no longer needed. Can you test it?